src/attr/_make.py

```diff
@@ -368,6 +368,8 @@ def wrap(cls):
                 cls = _add_pickle(cls)
         if slots is True:
             cls_dict = dict(cls.__dict__)
+            was_unhashable = ('__hash__' in cls_dict and
```

Comment:
Please add a human-readable comment what this means exactly…otherwise we’ll stumble over it every time we try to understand the code. :)

Also please `"` 🐶